app/controllers/api/projects.php

```diff
@@ -111,6 +111,7 @@
             'personalDataCheck' => false,
             'mockNumbers' => [],
             'sessionAlerts' => false,
+            'teamHideSensitiveFields' => false,
```

Comment:
```suggestion
              'teamsShowSensitiveFields' => true,
```
To make it easier logically, it's better to avoid inverse logic.

Reply:
Sure although usage becomes awkward in cases where it's null